vue/sbc-common-components/src/components/SbcFeeSummary.vue

```diff
@@ -1,163 +1,152 @@
 <template>
-    <v-card>
-        <header class="font-weight-bold px-3 py-3">
-            <slot name="header">Fee Summary</slot>
-        </header>
-
-        <div  v-show="fetchError">
-            <v-alert
-                    :value="true"
-                    color="error"
-                    icon="warning"
-                    outlined
-            >{{fetchError}}
-            </v-alert>
-
-        </div>
-        <v-slide-y-transition group tag="ul" class="fee-list"  v-show="!fetchError">
-            <li class="container fee-list__item" v-show="lineItem.fee" v-for="lineItem in fees" :key="lineItem.filingType" >
-                <div class="fee-list__item-name">{{lineItem.filingType}}</div>
-                <div class="fee-list__item-value">{{lineItem.fee | currency }}</div>
-            </li>
-        </v-slide-y-transition>
-        <div class="container fee-total" v-show="!fetchError">
-            <div class="fee-total__name">Total Fees</div>
-            <div class="fee-total__currency">CAD</div>
-            <div class="fee-total__value">
-                <v-slide-y-reverse-transition name="slide" mode="out-in">
-                    <div>{{totalFilingFees | currency}}</div>
-                </v-slide-y-reverse-transition>
-            </div>
-        </div>
-    </v-card>
+  <v-card>
+    <header class="font-weight-bold px-3 py-3">
+      <slot name="header">Fee Summary</slot>
+    </header>
+
+    <div v-show="fetchError">
+      <v-alert
+              :value="true"
+              color="error"
+              icon="warning"
+              outlined
+      >{{fetchError}}
+      </v-alert>
+
+    </div>
+    <v-slide-y-transition group tag="ul" class="fee-list" v-show="!fetchError">
```

Comment:
Although this was existing code, I recommend removing the v-slide-y-transition as this animation looks really weird when you Cease and then Undo a director.

Reply:
Does not reproduce for me atm.